ansible/group_vars/sonic/variables

```diff
@@ -25,8 +25,9 @@ mellanox_spc2_hwskus: [ 'ACS-MSN3700', 'ACS-MSN3700C', 'ACS-MSN3800', 'Mellanox-
 mellanox_spc3_hwskus: [ 'ACS-MSN4700', 'Mellanox-SN4700-O28', 'Mellanox-SN4700-O32', 'ACS-MSN4600', 'ACS-MSN4600C', 'ACS-MSN4410', 'Mellanox-SN4600C-D112C8', 'Mellanox-SN4600C-C64', 'Mellanox-SN4700-O8C48', 'Mellanox-SN4700-O8V48', 'ACS-SN4280', 'Mellanox-SN4280-O28', 'Mellanox-SN4280-O8C40', 'Mellanox-SN4280-C48', 'Mellanox-SN4280-O8V40', 'Mellanox-SN4280-O8C80', 'Mellanox-SN4700-V64', 'Mellanox-SN4700-O32']
 mellanox_spc4_hwskus: [ 'ACS-SN5600' , 'Mellanox-SN5600-V256', 'Mellanox-SN5600-C256S1', 'Mellanox-SN5600-C224O8',
                         'Mellanox-SN5610N-C256S2', 'Mellanox-SN5610N-C224O8']
-mellanox_spc5_hwskus: [ 'Mellanox-SN5640-C512S2', 'Mellanox-SN5640-C448O16']
-mellanox_hwskus: "{{ mellanox_spc1_hwskus + mellanox_spc2_hwskus + mellanox_spc3_hwskus + mellanox_spc4_hwskus + mellanox_spc5_hwskus + mellanox_spc5_hwskus }}"
+mellanox_spc5_hwskus: [ Mellanox-SN5640-C512S2', 'Mellanox-SN5640-C448O16']
```

Comment:
Looks like there is a missing leading '.

Will go with this PR: https://github.com/sonic-net/sonic-mgmt/pull/18921